app/controllers/copies_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CopiesController < ApplicationController
+  def index
+    @book = Book.find(params[:book_id])
+    @copies = @book.copies
+  end
+  def show
+    @copy = params
```

Comment:
Could you link to how this works?

Reply:
I don't think it does actually. Will change it now